### ISSUE TITLE
Profile header menu

### ISSUE THREAD

**Issue by matthewgallo** [carbon-design-system/ibm-products#1325]
# Profile header menu

## Brief description
Users rely on the profile menu to find information related to their account, the product they are using and their active session. This pattern describes the types of content, the layout and the interactions to be used with the profile menu in the global toolbar.

## Details

[Link to Design](https://pages.github.ibm.com/cdai-design/pal/patterns/profile-header-menu/)

### Links to other materials

![image](https://user-images.githubusercontent.com/10215203/136969663-5ddd37b8-664e-4048-bdf7-828628f77941.png)

## Design Maintainer(s)
Rami Alayan
slack: @rmalayan
email: rmalayan@us.ibm.com

## Tasks

Before starting work on this epic, please review and complete the following.

- [ ] Initial review of design/existing code

### Working in Cloud and Cognitive package

- [ ] Have you reviewed our
      [contribution guidelines](https://github.com/carbon-design-system/ibm-cloud-cognitive/blob/master/.github/CONTRIBUTING.md)?
- [ ] Have you noted our
      [code guidelines](https://github.com/carbon-design-system/ibm-cloud-cognitive/blob/master/docs/CODE_GUIDELINES.md)?
- [ ] Did you use the CLI to scaffold your new component?

**Comment by elycheea** [comment]
Moving this to on hold — saw in the PAL repo that there are likely updates coming to this pattern.

**Comment by elycheea** [comment]
Checked with Gentry and it looks as though this will be replaced with the profile menu from the CPFX team. Will move this back to our queue once that’s been updated.
